docs/logs/using.asciidoc

```diff
@@ -1,112 +1,86 @@
 [role="xpack"]
 [[xpack-logs-using]]
-== Using the Logs UI
+== Using the Logs app
+Use the Logs app in {kib} to explore and filter your logs in real time.
 
-Customize the Logs UI to focus on the data you want to see and control how you see it.
+You can customize the output to focus on the data you want to see and to control how you see it.
+You can also view related application traces or uptime information where available.
 
 [role="screenshot"]
-image::logs/images/logs-console.png[Log Console in Kibana]
+image::logs/images/logs-console.png[Logs Console in Kibana]
+// ++ Update this
 
 [float]
 [[logs-search]]
-=== Use the power of Search
-The Search bar is always available. Use it to perform adhoc and structured
-searches by taking advantage of the <<kuery-query>> with autocompletion
-suggestions.
+=== Use the power of search
 
-[role="screenshot"]
-image::logs/images/logs-usage-query-bar.png[Logs query bar]
+Use the search bar to perform ad hoc searches for specific text.
+You can also create structured queries using {kibana-ref}/kuery-query.html[Kibana Query Language].
+For example, enter `host.hostname : "host1"` to see only the information for `host1`.
+// ++ this isn't quite the same as the corresponding infrastructure description now.
 
 [float]
 [[logs-configure-source]]
-=== Adapt to your log source
-Using a custom index pattern to store the log entries, want to limit the
-entries presented in a space or change the fields displayed in the columns? Use
-<<xpack-logs-configuring,configure source>> to change the index pattern and
-other settings.
-
-[role="screenshot"]
-image::logs/images/logs-usage-column-headers.png[Logs column headers]
+=== Configure the data to use for your logs
+Are you using a custom index pattern to store the log entries?
+Do you want to limit the entries shown or change the fields displayed in the columns?
+If so, <<xpack-logs-configuring, configure the logs source data>> to change the index pattern and other settings.
 
 [float]
 [[logs-time]]
-=== Jump to a specific time period
-Use the time selector to focus on a specific timeframe.
+=== Specify the time and date
 
-[role="screenshot"]
-image::logs/images/logs-usage-time-picker.png[Logs time selector]
-
-To quickly jump to a nearby point in time, click on the timeline to the right.
-
-[role="screenshot"]
-image::logs/images/logs-usage-timeline.png[Logs timeline]
+Click the time selector image:logs/images/logs-time-selector.png[time selector icon] to choose the timeframe for the logs.
+Log entries for the time you specify appear in the middle of the page, with the earlier entries above and the later entries below.
 
+To quickly jump to a nearby point in time, click the minimap timeline to the right.
+// ++ what's this thing called? It's minimap in the UI. Would timeline be better?
```

Comment:
Good question. I feel timeline may be understood better by users, would be interested to hear others' thoughts.

Reply:
OK. I'd also like see what @gchaps thinks (she's the guru for UI text in Kibana, and maintains a list of Kibana UI components and their agreed names). Once we agree a name, minimap, timeline or otherwise, it should probably be probably another item to add to her list. 

I'll leave as is for now, since my carefully chosen words include both terms :-)